content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst

```diff
@@ -33,117 +34,95 @@ SEPA Direct Debit Mandates
 Create a mandate
 ----------------
 
-The SEPA Direct Debit Mandate is the document that your customers sign to authorize you to collect
+The :abbr:`SDD` Mandate is the document that your customers sign to authorize you to collect
 money directly from their bank accounts.
 
-To create a new mandate, go to :menuselection:`Accounting --> Customers --> Direct Debit Mandates`,
-click on *Create*, and fill out the form.
-
-.. image:: batch_sdd/batch-sdd-mandate-form.png
-   :align: center
-   :alt: Fill out SEPA Direct Debit in Odoo Accounting
-
-Export the PDF file by clicking on *Print*. It is then up to your customer to sign this document.
-Once done, upload the signed file in the **Original Document** field, and click on *Validate* to
-start running the mandate.
+To create a new mandate, go to :menuselection:`Accounting app --> Customers --> Direct Debit Mandates`,
+click on :guilabel:`Create`, and fill out the form.
+Export the PDF file by clicking on :guilabel:`Print`. It is then up to your customer to sign this
```

Comment:
Also, don't force a line break in RST when you don't intend to break the line in HTML.